README.md

```diff
@@ -1,4 +1,4 @@
-# CodeIgniter Rest Server
+# CodeIgniter Rest Server with JWT Authentication
```

Comment:
Hi  chriskacerguis,
its my first time to form a repository ,i will fix it.
thank you